README.md

```diff
@@ -1,61 +1,147 @@
+
+         /home/hadoop/shunting-yard/bin/replicator.sh --config=/path/to/config/file.yml
 
-### How to install the emitter - SOURCE
+### EMR
+If you are planning to run Shunting Yard on EMR you will need to set up the EMR classpath by exporting the following environment variables before calling the `bin/replicator.sh` script:
 
-On the source cluster, copy the file respective emitter JAR to _/usr/lib/hive/lib/_ and configure _/etc/hive/conf/hive-site.xml_ with the properties required by the listener to talk to the messaging infrastructure:
+         export HCAT_LIB=/usr/lib/hive-hcatalog/share/hcatalog/
+         export HIVE_LIB=/usr/lib/hive/lib/
 
-#### SQS
-    hive.metastore.event.listeners = com.hotels.shunting.yard.event.emitter.sqs.listener.SqsMetaStoreEventListener
-    com.hotels.shunting.yard.event.emitter.sqs.queue = https://sqs.<region>.amazonaws.com/<account-id>/<topic-name>-queue.fifo
-    com.hotels.shunting.yard.event.emitter.sqs.group.id = <group-id>
+Note that the paths above are correct as of when this document was last updated but may differ across EMR versions. Refer to the [EMR release guide](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-components.html) for more up to date information if necessary.
 
-Once this is done restart Hive:
+## Configuring Shunting Yard
 
-    sudo restart hive-hcatalog-server
+The examples below all demonstrate configuration using YAML and provide fragments covering the most common use cases that should be useful as a basis for building your own configuration. A full configuration reference is provided in the following sections.
 
-### How to install the receiver - TARGET
+### Configuring source, replica and SQS queue
 
-On the target cluster, download and install the latests version of Circus Train and set the `CIRCUS_TRAIN_HOME` environment variable:
+The YAML fragment below shows some common options for setting up the base source (where data is coming from), replica (where data is going to) and the SQS queue to read hive events from.
 
-    export CIRCUS_TRAIN_HOME=/home/hadoop/circus-train-<circus-train-version>
+    source-catalog:
+      name: source-cluster
+      hive-metastore-uris: thrift://emr-master-1.compute.amazonaws.com:9083
+    replica-catalog:
+      name: replica-cluster
+      hive-metastore-uris: thrift://emr-master-2.compute.amazonaws.com:9083
+    event-receiver:
+      configuration-properties:
+        com.hotels.shunting.yard.event.receiver.sqs.queue: https://sqs.us-west-2.amazonaws.com/123456789/sqs-queue
+        com.hotels.shunting.yard.event.receiver.sqs.wait.time.seconds: 20
+    source-table-filter:
+      table-names:
+        - ...
+    table-replications:
+      ...
 
-Export the location of the Hive libraries:
+### Selecting tables to monitor
 
-    export HIVE_LIB=/usr/lib/hive/lib/
-    export HCAT_LIB=/usr/lib/hive-hcatalog/share/hcatalog/
+The YAML fragment below shows how to select the tables to be monitored by Shunting Yard.
```

Comment:
Spell it out a bit more, something like "Shunting Yard by default will replicate all tables in a source Hive Metastore but the user can choose to limit this to a subset of tables by using a source table filter. For example to only replicate tables names `test_table_1` and `test_table_2` you would configure a `source-table-filter` like so:"